LEAF_Request_Portal/js/formGrid.js

```diff
@@ -992,7 +986,14 @@ var LeafFormGrid = function (containerID, options) {
         'dynicons/?img=x-office-spreadsheet.svg&w=16" alt="" /> Export</button>'
     );
 
-    $("#" + prefixID + "getExcel").on("click", function () {
+    $("#" + prefixID + "getExcel").on("click", async function () {
+      // get indicator formats in case they need special handling (e.g. dates)
```

Comment:
Are all the formIDs present in the report available at this point by any chance?

Reply:
The formIDs aren't guaranteed to be here since it'd require the user to use a Type filter.

All the indicatorIDs are present at this point, but I'll leave expanding the API for a separate story.